### PR TITLE
PMM-7400: Initial refresh of PMM.cd 

### PR DESCRIPTION
**Pull request by Sudokamikaze**
TestURL: https://pmm2.cd.percona.com/

AWS-Staging-Start result: https://pmm2.cd.percona.com/job/aws-staging-start/
PMM-testsuite result: https://pmm2.cd.percona.com/job/pmm2-testsuite/5095/

Please notice, all jobs are **disabled**, not to create any interference with main pmm jenkins
Feel free to enable them before any tests
Master has 0 executors so that pmm-submodule pipeline won't accidentally run

Features implemented:
* All latest features which we implemented in our other stacks were ported(Certbot, Log backup, etc)
* AWS Workers IaC, most options ported from original Jenkins but highly modified:
CentOS 6 and 7 latest images
AMZN 2 Linux latest images
* PMM Jenkins stack refresh:
Drop of NFS. As we've agreed to use EBS volume
Replacement of all "service" occurrences to use corresponding utility -- systemctl
Openresty instead of NGINX to evaluate usage of manage-nginx-allow-list job (manage-nginx-allow-list job idea dropped, it will be unused as PMM doesn't use .ymls for jobs. operesty left in place)
SSL Backup and restore when Jenkins was restarted/redeployed
SSH securing, by default it uses vulnerable options, audited & implemented by `https://github.com/arthepsy/ssh-audit`

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]

File: IaC/pmm.cd/init.groovy.d/matrix.groovy [hunk 1]
```diff
@@ -148,6 +148,7 @@ authz_strategy_config = [
         'percona*PMM': ['Overall Read','Agent Build','Agent Configure','Agent Connect','Agent Create','Agent Delete','Agent Disconnect','Job Build','Job Cancel','Job Configure','Job Create','Job Delete','Job Discover','Job Move','Job Read','Job Workspace','Run Delete','Run Update','Run Replay','View Configure','View Create','View Delete','View Read','SCM Tag'],
         'percona*ext-victoriametrics': ['Overall Read','Job Build','Job Cancel','Job Read','Run Replay','Run Update'],
```
Comment (atymchuk) [inline IaC/pmm.cd/init.groovy.d/matrix.groovy:149]: `percona*ext-victoriametrics` can be deleted

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: IaC/pmm.cd/init.groovy.d/matrix.groovy (hunk 1 above)
Comment (Sudokamikaze) [reply IaC/pmm.cd/init.groovy.d/matrix.groovy]: @atymchuk Please re-review, updated in  ecb8fbd

File: IaC/pmm.cd/JenkinsStack.yml [hunk 2]
```diff
@@ -0,0 +1,839 @@
...
+              			    proxy_set_header        X-Forwarded-Proto \$scheme;
+              			    proxy_redirect http:// https://;
+              			    proxy_pass              http://jenkins;
+              			    # Required for new HTTP-based CLI
+              			    proxy_http_version 1.1;
+              			    proxy_request_buffering off;
+              			    proxy_buffering off; # Required for HTTP-based CLI to work over SSL
+              			    # workaround for https://issues.jenkins-ci.org/browse/JENKINS-45651
+              			    add_header 'X-SSH-Endpoint' '$JENKINS_HOST:50022' always;
+                                    rewrite ^/$ $scheme://$server_name/blue/pipelines/ last;
```
Comment (atymchuk) [inline IaC/pmm.cd/JenkinsStack.yml:762]: Broken indentation :)

Comment (Sudokamikaze) [reply IaC/pmm.cd/JenkinsStack.yml:762]: Thank you, my local IDE creates this, will fix by holy vim

**Review by atymchuk — APPROVED** [review]

**Pull request merged**